.github/workflows/releaser.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    tags:
+      # so a vX.Y.Z-test1 doesn't trigger build
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-pre*'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # pin@v3
+        with:
+          fetch-depth: 0
+      - uses: docker/setup-qemu-action@5927c834f5b4fdf503fca6f4c7eccda82949e1ee # pin@v1
+      - uses: docker/setup-buildx-action@4fd812986e6c8c2a69e18311145f9371337f27d4 # pin@v1
+      - name: Set up Go
+        uses: actions/setup-go@v5 # pin@v3
+        with:
+          go-version: '1.22'
+          check-latest: true
```

Comment:
I think providing the second implies the first one is useless

Reply:
no it stays on 1.22 (1.21 before) even when 1.23 ships